README.md

```diff
@@ -156,19 +156,19 @@ complete(myEntity).ok().done();
 The following is an example
 
 ```
-var sdk = require('kinvey-backend-sdk');
-var service = sdk.service(function(err, service) {
-  var dataLink = service.dataLink;   // gets the datalink object from the service
-  var notImplementedHandler = function(request, complete) {
+let sdk = require('kinvey-backend-sdk');
+let service = sdk.service((err, service) => {
+  let dataLink = service.dataLink;   // gets the datalink object from the service
+  let notImplementedHandler = (request, complete) => {
     complete("These methods are not implemented").notImplemented().done();
   };
 
-  var getRecordById = function(request, complete) {
-    var entityId = request.entityId;
-    var entity;
+  let getRecordById = (request, complete) => {
+    let entityId = request.entityId;
+    let entity = null;
 
     // Do some logic to get the entity id from the remote data store
-    // Assume that data is retrieved and stored in "entity" variable
+    // Assume that data is retrieved and stored in "entity" letiable
```

Comment:
I changed this but just so you know, that this was a find/replace victim. I don't actually think the word 'letiable' is a thing 😄